media-libs/zint/zint-2.14.0.ebuild

```
@@ -0,0 +1,60 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop xdg
+
+DESCRIPTION="A barcode encoding library supporting over 50 symbologies."
```

Avoid "A..." and also terminating full stops.
```suggestion
DESCRIPTION="Barcode encoding library supporting over 50 symbologies"
```